rustls/src/msgs/message_test.rs

```diff
@@ -72,7 +68,7 @@ fn can_read_safari_client_hello() {
     let mut rd = Reader::init(bytes);
     let m = OutboundOpaqueMessage::read(&mut rd).unwrap();
     println!("m = {:?}", m);
-    assert!(Message::try_from(m.into_plain_message()).is_err());
+    assert!(Message::try_from(m).is_err());
```

Comment:
This doesn't make precise sense either. When we're testing decoding a message, that should be using `InboundOpaqueMessage::read`. `OutboundOpaqueMessage::read` should not exist, because there is no functional reason for a TLS library to decode messages that is generating.